todo_app/item.py

```diff
@@ -0,0 +1,19 @@
+class Item():
+    def __init__(self, id, name, description, due_date, id_list, id_board, status='To Do'):
+        self.id = id
+        self.name = name
+        self.description = description
+        self.due_date = due_date
+        self.id_list = id_list
+        self.id_board = id_board
+        self.status = status
+
+class TrelloBoard():
```

Comment:
Minor, but I'd generally suggest having separate files even when the classes are really small